packages/x-data-grid-premium/src/tests/clipboard.DataGridPremium.test.tsx

```diff
@@ -343,10 +343,10 @@ describe('<DataGridPremium /> - Clipboard', () => {
 
         await waitFor(() => {
           expect(getCell(3, 3).textContent).to.equal('p33');
-          expect(getCell(6, 2).textContent).to.equal('p62');
-          expect(getCell(7, 1).textContent).to.equal('p71');
-          expect(getCell(7, 3).textContent).to.equal('p73');
         });
+        expect(getCell(6, 2).textContent).to.equal('p62');
```

Comment:
waitFor allows only one `expect` to be inside of it and it makes sense—anything after it can be outside of the awaited `waitFor`. 👌